styles/modules/modals/_editListing.scss

```diff
@@ -39,6 +39,10 @@
 
     .tabContent {
       .photoUploadSection {
+        h2 {
+          display: inline-block;
```

Comment:
During an upload, if that h2 isn't an inline-block, then the "uploading" status gets a scrollbar.

![image](https://cloud.githubusercontent.com/assets/794517/21781823/f41af7ca-d665-11e6-899f-e770685bf5ef.png)

I know I could probably re-work this to use flex, but I'll be gosh darned if I'm going to use flex for ever little think... lol!